docs/docs/en/development/backend/spi/alert.md

```diff
@@ -22,7 +22,7 @@ If you don't care about its internal design, but simply want to know how to deve
 
 * dolphinscheduler-alert-plugins
 
-  This module is currently a plug-in provided by us, such as Email, DingTalk, Script, etc.
+  This module is currently a plug-in provided by us, and now we have supported ten plug-ins, such as Email, DingTalk, Script, etc.
```

Comment:
I prefer to keep it in the old content, if we specify the number of our plugins, we should change this number every time we add plugins. I think most people will forget to modify it, WDYT？
```suggestion
  This module is currently a plug-in provided by us, such as Email, DingTalk, Script, etc.
```

Reply:
The reason why I changed this is I think now we have supported a lot of different plug-ins and this is an advantage so I want to show them out. 

When a new user just looking around, Maybe some plug-ins are not very common but we supported them so he or she will be interested in our product, Maybe we can support a lot of different plug-ins is a reason for him or her to choose our product. 

So I think the new content is better, The only question is how to explain the num in English. 

In Chinese doc, I use broad expressions, not a clear number, you can understand it in ten or more. Maybe in English doc changing ten to dozens is better? We use broad expressions so we don’t need to change this word when we add a new plug-in.

```This module is currently a plug-in provided by us, and now we have supported dozens of plug-ins, such as Email, DingTalk, Script, etc.```

What do you think?

Reply:
> The reason why I changed this is I think now we have supported a lot of different plug-ins and this is an advantage so I want to show them out.

Sound good, So we need to use a vague amount, not a specific value, the word `we have supported dozens of plug-ins` looks good!

Reply:
Could you change the content?

Reply:
Done